v2/api/dbformysql/v1api20211201previewstorage/structure.txt

```diff
@@ -0,0 +1,37 @@
+// Code generated by azure-service-operator-codegen. DO NOT EDIT.
+github.com/Azure/azure-service-operator/v2/api/dbformysql/v1api20211201previewstorage
+├── APIVersion: Enum (1 value)
+│   └── "2021-12-01-preview"
+├── FlexibleServersAdministrator: Resource
+│   ├── Owner: github.com/Azure/azure-service-operator/v2/api/dbformysql/v1api20211201preview.FlexibleServer
+│   ├── Spec: Object (10 properties)
+│   │   ├── AdministratorType: *string
+│   │   ├── IdentityResourceReference: *genruntime.ResourceReference
+│   │   ├── Login: *string
+│   │   ├── OriginalVersion: string
+│   │   ├── Owner: *genruntime.KnownResourceReference
+│   │   ├── PropertyBag: genruntime.PropertyBag
+│   │   ├── Sid: *string
+│   │   ├── SidFromConfig: *genruntime.ConfigMapReference
+│   │   ├── TenantId: *string
+│   │   └── TenantIdFromConfig: *genruntime.ConfigMapReference
+│   └── Status: Object (11 properties)
+│       ├── AdministratorType: *string
+│       ├── Conditions: conditions.Condition[]
+│       ├── Id: *string
+│       ├── IdentityResourceId: *string
+│       ├── Login: *string
+│       ├── Name: *string
+│       ├── PropertyBag: genruntime.PropertyBag
+│       ├── Sid: *string
+│       ├── SystemData: *Object (7 properties)
+│       │   ├── CreatedAt: *string
+│       │   ├── CreatedBy: *string
+│       │   ├── CreatedByType: *string
+│       │   ├── LastModifiedAt: *string
+│       │   ├── LastModifiedBy: *string
+│       │   ├── LastModifiedByType: *string
+│       │   └── PropertyBag: genruntime.PropertyBag
+│       ├── TenantId: *string
+│       └── Type: *string
+└── augmentConversionForSystemData_STATUS: Interface
```

Comment:
Why do we have an `augmentConversion` interface for only STATUS?

Reply:
The `SystemData` property only exists on the `Status` type. All of the other properties on the `Spec` are primitives or `genruntime` types.

So I think this is correct, the only complex child type is `SystemData` and it only exists on status, so that's the only complex child we have `augmentConversionFor` interfaces for?